packages/relay/src/index.ts

```diff
@@ -56,6 +51,6 @@ async function main() {
     const routes = await fs.promises.readdir(routeDir)
     for (const routeFile of routes) {
         const { default: route } = await import(path.join(routeDir, routeFile))
-        route(app, synchronizer._db, synchronizer)
+        route(app, prover, db, synchronizer)
```

Comment:
Here it routes 4 parameters (`app`, `prover`, `db`, and `synchronizer`)
so other routers should take 4 inputs
```ts
export default (app: Express, prover: Prover, db: DB, synchronizer: Synchronizer) => {
```